tailwind.config.js

```diff
@@ -0,0 +1,119 @@
+/** @type {import('tailwindcss').Config} */
+export default {
+  content: ['./index.html', './src/**/*.{js,ts,jsx,tsx}'],
+  theme: {
+    extend: {
+      screens: {
+        custom: '1450px',
+        customSmall: '400px'
```

Comment:
add intermediate screen-sizes, 680px, 920px.
you've used 'custom' breakpoint everywhere, but is too wide.